ui/app/router.js

```diff
@@ -15,9 +15,8 @@ Router.map(function() {
       this.route('logout');
       this.mount('open-api-explorer', { path: '/api-explorer' });
       this.route('license');
-      this.route('metrics', function() {
+      this.route('clients', function() {
```

Comment:
does changing the route names affect the picker at all on the Status menu?  E.g. can you still navigate to the page the same way as before?

Reply:
I updated all the references which were pointing to metrics route, so it will work fine.